armada/handlers/armada.py

```diff
@@ -198,3 +202,26 @@ def show_diff(self, chart, installed_chart,
                 LOG.debug(line)
 
         return (len(chart_diff) > 0) or (len(values_diff) > 0)
+
+    def wait_until_deployed(self):
+        #
+        # TODO: configure command line arg to call function
+        #       command line arg for timeout 
+        #
+        os_pods = self.tiller.k8s.get_all_pods()
+        for pod in os_pods.items:
+            name = pod.metadata.name
+            status = pod.status.conditions[0].status
+            if status != "True":
+                LOG.info("{}-{}".format(name, status))
+                # wait for pod status to be True
+                #  check every second until timeout? 
+                default_timeout = 10
```

Comment:
@alanmeadows @gardlt in regards to specifying a timeout for each chart, should this be a field in the yaml?